Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/DirectionalLight.azsli

```diff
@@ -50,21 +68,20 @@ void ApplyDirectionalLights(Surface surface, inout LightingData lightingData)
         // [GFX TODO][ATOM-2012] care of multiple directional light
         // Currently shadow check is done only for index == shadowIndex.
         float currentLitRatio = 1.0f;
-        float currentBackShadowRatio = 1.0f;
-        if (o_enableShadows)
+        float currentTransmissionDistance = -1.0f;
+
+        if (o_enableShadows && index == shadowIndex)
         {
-            currentLitRatio = (index == shadowIndex) ? litRatio : 1.;
-            
-            currentBackShadowRatio = 1.0 - currentLitRatio;
-            if (o_transmission_mode == TransmissionMode::ThickObject)
-            {
-                currentBackShadowRatio = (index == shadowIndex) ?  backShadowRatio : 0.;
-            }
+            // Add contribution only if current directional light is the active one for shadows
+            currentLitRatio = litRatio;
+            currentTransmissionDistance = transmissionDistance;
         }
+
+        // Transmission contribution
+        lightingData.translucentBackLighting += GetBackLighting(surface, lightingData, light.m_rgbIntensityLux, dirToLight, currentTransmissionDistance, camToSurfDist/2.0);
```

Comment:
Why do we divide camToSurfDist by 2 here? Can you add a comment explaining?

Reply:
I thought it would make sense to attenuate the effect for different types of lights at different speeds, but after playing a bit with it, there's no big difference. Thanks for pointing out.